src/components/utils/http.js

```diff
@@ -22,7 +22,8 @@ const errorMessage = (status) => {
   }
 }
 
-const handleError = (status, callback) => {
+const handleError = ({status, response}, callback) => {
+  console.error(status, response)
   const message = errorMessage(status)
   Tracker.sendError(message)
```

Comment:
I agree with Henry, that it's fine to pass the error along. It's one less layer of indirection.
Also, don't worry too much about this logging. We have accomplished the scope of the task which was to convert the protocols, if there is still stuff you guys want to work on it can be added to another issue.